content/master/concepts/claim-server-side-apply.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Claim Server-Side Apply
```

Comment:
I prefer keeping alpha features out of the main flow - not mixed in. This makes it _really_ clear that it's separate and alpha.

@plumbis FWIW about 60% of this preference is because AFAIK our nice "this feature is alpha, it was introduced in version X" widget only works as front matter for a whole page. If I could do it on a heading I'd be more amenable to merging alpha features into main docs flows as a separate heading.

```markdown
state: alpha
alphaVersion: "1.15"
```